src/test/java/org/opensearch/security/TracingTests.java

```diff
@@ -239,16 +237,6 @@ public void testHTTPTraceNoSource() throws Exception {
 
     @Test
     public void testHTTPSingle() throws Exception {
-
-        Thread.setDefaultUncaughtExceptionHandler(new UncaughtExceptionHandler() {
```

Comment:
out of curiosity, why did we remove this handler?

Reply:
UncaughtExceptionHandlers can muck with Java internals in ways that make them unpredictable - when we were switched to use Random test runner from OpenSearch's test infrastructure it applied a rule warning about this behavior that is typically bad (but not always).  To get the tests to run without errors I had to remove these calls.

When I reviewed these tests this behavior was not necessary and likely made the tests slower and flaky